src/modules/rendercomment.js

```diff
@@ -0,0 +1,51 @@
+const commUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/CvrqAzoVr9PCSxK9Vq6U/comments';
+
+export const renderComment = async (id, name, comment) => {
+  const response = await fetch(`${commUrl}`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ item_id: id, username: name, comment }),
+  });
+  return response;
+};
+
+const fetchComment = async (id) => {
+  const reponseComment = await fetch(`${commUrl}?item_id=${id}`);
+  const itemData = await reponseComment.json();
+
+  if (!reponseComment.ok) {
+    return null;
+  }
+  return itemData;
+};
+
+/* eslint-disable */
+const createComments = ({ comment, username, creation_date }) => {
+  const commentli = document.createElement('li');
+  commentli.className = 'comment-li';
+  commentli.innerHTML = `
+   <span>${creation_date}: </span><span>${username}: </span><span>${comment}</span>
+  `;
+  return commentli;
+};
+/* eslint-enable */
```

Comment:
- [ ] Kindly enable the linters here too. 🤓

Reply:
On line 25 we store the properties of the object obtained by API so we cannot change the name, since we cannot change the name so it gives a linter error for "creation_date", so that is why we disabled the linter there.

Reply:
[Tried this](https://github.com/Wahaj-Ali/tv-hub/commit/f46d3b121b02bafa64523762b6ef5604f60e8db5)